CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ SET(CMAKE_C_FLAGS "-std=c99 -Werror=implicit-function-declaration ${CMAKE_C_FLAG
 ENDIF (NOT WIN32)
 IF(CUDA_HAS_FP16 OR NOT ${CUDA_VERSION} LESS 7.5)
   SET(CMAKE_C_FLAGS "-DCUDA_HAS_FP16=1 ${CMAKE_C_FLAGS}")
+  SET(CMAKE_C_FLAGS "-DREDUCED_TYPE_SET=1 ${CMAKE_C_FLAGS}")
```

Comment:
Thanks for catching this: yes it was supposed to be THC_REDUCED_TYPE_SET, for testing purposes only.